packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -447,16 +447,27 @@ export class WorkspaceRootUriAwareCommandHandler extends UriAwareCommandHandler<
         super(selectionService, handler);
     }
 
-    protected getUri(): URI | undefined {
-        const uri = super.getUri();
-        if (this.workspaceService.isMultiRootWorkspaceEnabled) {
-            return uri;
-        }
+    // tslint:disable-next-line: no-any
+    public isEnabled(...args: any[]): boolean {
+        return super.isEnabled(args) && !!this.workspaceService.tryGetRoots().length;
```

Comment:
`...args`, otherwise only one array is passed as one arg, please fix in other places

Also should not it be `||`, since roots are only fallback, and not mandatory if super check is successful?

Reply:
ok